cores/rp2040/RP2040Support.h

```diff
@@ -314,6 +315,18 @@ class RP2040 {
         reboot();
     }
 
+    void wdt_begin(uint32_t delay_ms) {
+        watchdog_enable(delay_ms, 1);
+    }
+
+    void wdt_reset() {
+        watchdog_reset();
+    }
+
+    void sleep_until_ms(uint32_t ms) {
```

Comment:
This isn't needed.  `delay()` already does `sleep_ms`

Reply:
````
In file included from /home/runner/arduino_ide/hardware/pico/rp2040/cores/rp2040/Arduino.h:112,
                 from /home/runner/work/arduino-pico/arduino-pico/build.tmp/sketch/AnalogReadSerial.ino.cpp:1:
/home/runner/arduino_ide/hardware/pico/rp2040/cores/rp2040/RP2040Support.h:31:10: fatal error: pico/sleep.h: No such file or directory
   31 | #include <pico/sleep.h>
      |          ^~~~~~~~~~~~~~
compilation terminated.
exit status 1
````

`pico/time.h` has the `sleep_ms` call definition BTW.  If you do need `pico/sleep.h` then we need to update `tools/libpico/Cmakefile.txt`, rebuild libpico, and update the `lib/platform_inc` file.